src/modules/map/store/measurement.observer.js

```diff
@@ -0,0 +1,25 @@
+import { observe } from '../../../utils/storeUtils';
+import { addLayer, removeLayer } from './layers.action';
+
+/**
+ * Id of the layer used for measurement interaction
+ */
+export const MEASUREMENT_LAYER_ID = 'measurement_layer';
+
+export const registerMeasurementObserver = (store) => {
```

Comment:
to mark this method as a implementation of a recipe or template for store-observer, i would suggest to rename the method to 'register'. This should help to identify and multiply the necessary steps to create new store-obsservers, similar to our usage of i18n-providers